#modes for the system
float32 inputType #joystick or ITCI
float32 figure_num #to show available inputs
float32 x
float32 y #x,y is position of ITCI on the itci image
float32 selectness #how close to selecting an input we are